### DISCUSSION TITLE
How do i enter japanese?

### DISCUSSION THREAD

**Discussion by KawakamiYuta** [neovide/neovide#1107] [Q&A]
I want to know how to input japanese on neovide.
maybe impossible?

i'm japanease like neovide graphics, but I'm having trouble entering Japanese.
please help me.

I use in Windows10.
i did the following.(on neovim-qt i can enter japanese on same operation)
first, start neovide and toggle insert mode,
then toggle inpurt source to japanese, and type japanese

**Comment by Kethku** [comment]
Can you try again? https://github.com/neovide/neovide/issues/243 some fixes have this have gone in recently

**Reply by KawakamiYuta** [reply]
thank you reply, but couldn't do it yet
I tried on release 0.6.0, 0.7.0

but, Behavior has changed from 0.5.0
on release 0.5.0:
when I switched to Japanese input, I couldn't input anything.

on release 0.6.0:
when I switched to Japanese input, still i can input but coudnt' show japanese.
if i typing "aiueo" in japanese input mode, i want display "あいうえお", but "aiueo?????" was displayed.("?????" is garbled character, and when copy it to other editor, it displayed as "aiueoあいうえお").

**Reply by Kethku** [reply]
Please try from main. Releases are old and not up to date. We Re pushing for a new release soon but are not there yet

**Reply by KawakamiYuta** [reply]
i success to enter japanease, thank you!!
